### PR TITLE
Pagination on status page, previous and next week

### PR DESCRIPTION
**Pull request by joecohens**
https://github.com/cachethq/Cachet/issues/148

### CODE REVIEW

**Comment by jbrooksuk** [comment]
Do we think that we should go back in weekly increments or go from showing the last 7 days, to everything?

File: app/views/index.blade.php [hunk 1]
```diff
@@ -26,7 +26,51 @@
     @endif --}}
 
     <h1>{{ Lang::get('cachet.past_incidents') }}</h1>
+
+    <?php
+        $today = Carbon::now();
+        $startDate = Carbon::now();
+
+        // Check if we have another starting point
+        if (Input::has('start_date')) {
```
Comment (GrahamCampbell) [inline app/views/index.blade.php]: This is bad.

Comment (GrahamCampbell) [inline app/views/index.blade.php]: The views should not know about http...

Comment (jbrooksuk) [inline app/views/index.blade.php]: I think @joecohens was just editing what we had originally. But yes, all of this should be within the controller.

Comment (joecohens) [inline app/views/index.blade.php]: Yea I know http should't be on the views, this was a proof of concept with what had been done before. This should be all moved to the controller. Ref https://status.github.com/messages

**Comment by GrahamCampbell** [comment]
This needs rebasing.

**Comment by joecohens** [comment]
Do you like this feature to reimplement on the new structure? https://github.com/cachethq/Cachet/pull/222

**Comment by GrahamCampbell** [comment]
Yes please.

**Comment by jbrooksuk** [comment]
I think we should have the last 7 days as default view, then a History button which shows all incidents?

**Comment by jbrooksuk** [comment]
Or all incidents and a last week button?

**Comment by joecohens** [comment]
@jbrooksuk Mmm I think someone using the status page for a long time and many incidents can be querying lots of rows don't you?

**Comment by joecohens** [comment]
@jbrooksuk Should I proceed?

**Comment by jbrooksuk** [comment]
@joecohens yeah, proceed, it'll be good to go back in time regardless :smile:

**Comment by joecohens** [comment]
:+1:

**Comment by GrahamCampbell** [comment]
Might be useful to do this after my next refactorings.

**Comment by joecohens** [comment]
Yes I think I'll wait a little bit more.

**Comment by GrahamCampbell** [comment]
My refactoring is done for now. :)

**Comment by jbrooksuk** [comment]
@joecohens rebase and we'll merge.

**Comment by joecohens** [comment]
Closed in favor of https://github.com/cachethq/Cachet/pull/257

**Pull request closed without merging**